### PR TITLE
[Sprite Lab] First new blocks for "Story Lab" (Heading and SubHeading)

### PR DESCRIPTION
**Pull request by mikeharv**
Alternative to:
- https://github.com/code-dot-org/code-dot-org/pull/48446

The difference here is we want to better isolate these experimental blocks from core Sprite Lab. We can add a block pool for "Story Lab" and move the commands and utils out to separate files that will be easier to track.

The new blocks will be maintainable on levelbuilder at `pools/storylab/blocks/` and are currently set to the "Story Lab" category for the purposes of editing level toolbox/start blocks. Block color was chosen to align to other "World" category blocks such as `show title screen`.
image

# Previous Description (Copied)

Create new `add heading` and `add subheading` blocks to the Sprite Lab pool. These blocks aren't visible to users yet. The heading and subheading will always each take up one line and the text will scale depending on how long they are. These are very similar to Poetry's `add title` and `add author` blocks, and the text scaling logic is copied and modified from those blocks.

## Screenshots

Screen Shot 2022-09-30 at 1 22 24 PM

Screen Shot 2022-09-30 at 1 22 59 PM

## Links

- jira ticket: [SL-275](https://codedotorg.atlassian.net/browse/SL-275)

## Follow-up work

We anticipate follow-up work, but the specifics are unknown at this time.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/storyLabUtils.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import {APP_WIDTH} from './constants';
+
+const OUTER_MARGIN = 50;
+const LINE_HEIGHT = 50;
+
+/**
+ * Draw a validation bar - a P5 shape comprised of a rectangle
```
Comment (molly-moen) [inline apps/src/p5lab/storyLabUtils.js]: update this comment?

**Review by molly-moen — APPROVED** [review]
Thanks for picking this up!

File: apps/src/p5lab/storyLabUtils.js (hunk 1 above)
Comment (mikeharv) [reply apps/src/p5lab/storyLabUtils.js]: Good catch - thank you!

**Comment by pablo-code-org** [comment]
i wanna add a note that I loved doing this PR :) 

It was very clean and encapsulated code with great variable names and easy to follow the math even without context. 

This PR followed really good coding practices I have seen in many books.

**Pull request merged**